src/mono/wasm/templates/templates/browser/main.js

```diff
@@ -1,5 +1,15 @@
 import { App } from './app-support.js'
 
 App.main = async function (applicationArguments) {
+    App.IMPORTS.window = {
+        location: {
+            href: () => globalThis.window.location.href
+        }
+    };
+
+    const exports = await App.MONO.mono_wasm_get_assembly_exports("browser.0.dll");
+    const text = exports[""].MyClass.Greeting();
```

Comment:
Fix for this is in https://github.com/dotnet/runtime/pull/72174

Reply:
The above is merged to main, so I made also fix here.
I'm not sure in which order they arrive to the workload. 
Today's rc.1 doesn't have it yet.
But it's one or the other, so it's better to have the nicer one already.